src/NativeScript/ObjC/ObjCMethodCall.h

```diff
@@ -46,7 +46,8 @@ class ObjCMethodCall : public FFICall {
 
 private:
     ObjCMethodCall(JSC::VM& vm, JSC::Structure* structure)
-        : Base(vm, structure) {
+        : Base(vm, structure)
+        , _hasErrorOutParameter{ false } {
```

Comment:
I could be wrong, but I believe we don't have to explicitly initialize this to `false`.